octree_creation_app/params.py

```diff
@@ -138,8 +138,8 @@ def build(cls, input_data: InputFile | dict) -> Self:
 
         return out
 
-    @staticmethod
-    def collect_input_from_dict(base_model: type[BaseModel], data: dict[str, Any]):
+    @classmethod
```

Comment:
not related. Committed by mistake?

Reply:
No, getting a mypy error regarding using super() instead of static method